sdk/core/abort-controller/src/AbortSignalLike.ts

```diff
@@ -10,6 +10,10 @@ export interface AbortSignalLike {
    * Indicates if the signal has already been aborted.
    */
   readonly aborted: boolean;
+  /**
+   * The reason why the signal was aborted.
+   */
+  readonly reason?: any;
```

Comment:
Might as well add the MDN docs just like they do with the DOM https://github.com/microsoft/TypeScript/blob/main/src/lib/dom.generated.d.ts#L2578C25-L2579